src/main/java/com/salaboy/conferences/agenda/controller/AgendaController.java

```diff
@@ -0,0 +1,86 @@
+package com.salaboy.conferences.agenda.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Strings;
+import com.salaboy.conferences.agenda.model.AgendaItem;
+import com.salaboy.conferences.agenda.model.Proposal;
+import com.salaboy.conferences.agenda.repository.AgendaItemRepository;
+import io.zeebe.client.api.response.ActivatedJob;
+import io.zeebe.client.api.worker.JobClient;
+import io.zeebe.spring.client.annotation.ZeebeWorker;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.Random;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@RestController
+@Slf4j
+@RequestMapping
+public class AgendaController {
+
+    private final AgendaItemRepository agendaItemRepository;
+    private final ObjectMapper objectMapper;
+
+    public AgendaController(AgendaItemRepository agendaItemRepository, ObjectMapper objectMapper) {
+        this.agendaItemRepository = agendaItemRepository;
+        this.objectMapper = objectMapper;
+    }
+
+    @PostMapping
+    public Mono<String> newAgendaItem(@RequestBody AgendaItem agendaItem) {
+
+        log.info("> New Agenda Item Received: " + agendaItem);
+
+        if (agendaItem.getTitle().contains("fail")) {
+            throw new IllegalStateException("Something went wrong with adding the Agenda Item: " + agendaItem);
+        }
+
+        Mono<String> result = agendaItemRepository.save(agendaItem)
```

Comment:
@mcruzdev do you really need to create `result` here? You can just add subscribe() to the end and return the value, without the need of creating the variable `result` right?

Reply:
@salaboy If I add subscribe() to the end I will receive a Disposable.
I need to subscribe() because the repository doesn't save the AgendaItem

Reply:
@salaboy I solve this problem, you are correct ;)